finality-aleph/src/network/manager/discovery.rs

```diff
@@ -3,40 +3,32 @@ use crate::{
         manager::{Authentication, Multiaddr, SessionHandler},
         DataCommand, PeerId, Protocol,
     },
-    NodeCount, NodeIndex, SessionId,
+    NodeIndex, SessionId,
 };
 use codec::{Decode, Encode};
-use log::{debug, warn};
+use log::{debug, trace, warn};
 use std::{
-    collections::{HashMap, HashSet},
+    collections::HashMap,
     time::{Duration, Instant},
 };
 
-/// How many nodes we should query about unknown authorities in one go.
-const NODES_TO_QUERY: usize = 2;
-
 /// Messages used for discovery and authentication.
 #[derive(Clone, Debug, PartialEq, Encode, Decode)]
 pub enum DiscoveryMessage {
     AuthenticationBroadcast(Authentication),
-    // Requests always contain own authentication, to avoid asymmetric trust.
-    Request(Vec<NodeIndex>, Authentication),
     // Always assumed to contain only authentications for one session.
```

Comment:
This comment needs to be adapted afaik , as it was more suitable for `Vec`

Reply:
I don't know if any comment is necessary now, since there are no assumptions for single authentication